kong/templates/nginx_kong.lua

```diff
@@ -66,7 +66,9 @@ upstream kong_upstream {
     balancer_by_lua_block {
         Kong.balancer()
     }
+> if upstream_keepalive > 0 then 
     keepalive ${{UPSTREAM_KEEPALIVE}};
+> end
```

Comment:
We would need you to provide a test for this behaviour. Nginx config template test can be found here: https://github.com/Kong/kong/blob/master/spec/01-unit/003-prefix_handler_spec.lua#L55